internal/tokenizer/tokenize_test.go

```diff
@@ -115,6 +115,45 @@ func TestTokenize(t *testing.T) {
 	}
 }
 
+func TestTokenizerLatin1AsUtf8(t *testing.T) {
+	content := []byte("th\xe5 filling") // `th� filling`
+	t.Logf("%v - %q", content, string(content))
+	tokens := Tokenize(content)
+	for i, token := range tokens {
+		t.Logf("token %d, %s", i+1, token)
+	}
+	require.Equal(t, 3, len(tokens))
+}
+
+func TestRegexpOnInvalidUtf8(t *testing.T) {
+	origContent := []struct {
+		bytes  []byte
+		tokens []string
+	}{
+		{[]byte("th\xe0 filling"), []string{"th", "filling"}},   // `th� filling`
```

Comment:
[optional] This is completely reasonable as you wrote it, but I often find it a little easier to read if the test input is given as a plain `string` and converted to `[]byte` at the point of use, e.g.,

```go
origContent := []struct {
    input string
    tokens []string
}{
   {"th\xe0 filling", []string{"th", "filling"}},
   …
}
…
input := []byte(content.input)
// … the rest of the test.
```

When there are a bunch of test cases, it adds up. It's up to you, though.

Reply:
Thank you for suggestion, addressed in f3ceaa6